### PR TITLE
Bunch of LOF Cards

### PR DESCRIPTION
**Pull request by sethhelman**
paz vizsla
unleash rage
kaadu
tip the scale

### CODE REVIEW

File: test/server/cards/05_LOF/events/TipTheScale.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+                // No choice here so no prompt, but the player still sees an opponent hand reveal
+                context.player1.clickCard(context.tipTheScale);
+                expect(context.player1).toHaveExactDisplayPromptCards({
+                    invalid: [context.sabineWren, context.battlefieldMarine, context.infernoFour]
+                });
+                context.player1.clickPrompt('Done');
+            });
+        });
+    });
```
Comment (adbronico) [inline test/server/cards/05_LOF/events/TipTheScale.spec.ts:63]: Can we add a test when the opponent has an empty hand? Should just be the event moving to discard and paying the costs, but then player 2 being active player

File: test/server/cards/05_LOF/units/Kaadu.spec.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['patrolling-vwing'],
+                        base: { card: 'echo-base', damage: 3 }
+                    }
+                });
+            });
+
+            it('should give its own ground unit +3/+3 and overwhelm', function () {
```
Comment (adbronico) [inline test/server/cards/05_LOF/units/Kaadu.spec.ts:22]: nit: update test to only say Overwhelm

File: test/server/cards/05_LOF/units/Kaadu.spec.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                expect(context.player1).toBeAbleToSelectExactly([context.gamorreanGuards, context.cartelSpacer]);
+                context.player1.clickCard(context.gamorreanGuards);
+
+                context.player2.clickPrompt('Pass');
+                context.player1.clickCard(context.gamorreanGuards);
+                context.player1.clickCard(context.battlefieldMarine);
+                expect(context.p2Base.damage).toBe(4);
+            });
+
+            it('should give friendly space unit +3/+3 and overwhelm', function () {
```
Comment (adbronico) [inline test/server/cards/05_LOF/units/Kaadu.spec.ts:35]: same with the test title here

File: server/game/cards/05_LOF/units/PazVizslaUnyieldingWarriror.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        return {
+            id: '3337614029',
+            internalName: 'paz-vizsla#unyielding-warrior',
+        };
+    }
+
+    protected override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'This unit gets +2/+0 for each damage on him',
+            ongoingEffect: AbilityHelper.ongoingEffects.modifyStats((target) => ({
```
Comment (adbronico) [inline server/game/cards/05_LOF/units/PazVizslaUnyieldingWarriror.ts:15]: Do we have any other spots that use target in this way? Just looking to maintain consistency

File: server/game/cards/05_LOF/units/PazVizslaUnyieldingWarriror.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+                power: +(target.damage * 2), hp: 0
```
Comment (adbronico) [inline server/game/cards/05_LOF/units/PazVizslaUnyieldingWarriror.ts:16]: Do we need the + here?

**Review by adbronico — CHANGES_REQUESTED** [review]

File: server/game/cards/05_LOF/units/PazVizslaUnyieldingWarriror.ts (hunk 4 above)
Comment (sethhelman) [reply server/game/cards/05_LOF/units/PazVizslaUnyieldingWarriror.ts:15]: this was copied from d'qar cargo frigate
